pysatMadrigal/instruments/madrigal_pandas.py

```diff
@@ -235,36 +241,39 @@ def list_files(tag, inst_id, data_path, kindat='', format_str=None,
     return out
 
 
-def download(date_array, tag='', inst_id='', data_path=None, user=None,
-             password=None, file_type='hdf5', kindat=''):
+def download(date_array, tag, inst_id, data_path, user=None, password=None,
+             file_type='hdf5', kindat=''):
     """Download data from Madrigal.
 
     Parameters
     ----------
     date_array : array-like
-        list of datetimes to download data for. The sequence of dates need not
+        List of datetimes to download data for. The sequence of dates need not
         be contiguous.
     tag : str
-        Madrigal Instrument code cast as a string. (default='')
+        Madrigal Instrument code cast as a string.
     inst_id : str
-        Satellite ID string identifier used for particular dataset. (default='')
+        Instrument ID used for particular dataset.
     data_path : str
-        Path to directory to download data to. (default=None)
-    user : str
-        User string input used for download. Provided by user and passed via
-        pysat. If an account is required for dowloads this routine here must
-        error if user not supplied. (default=None)
-    password : str
-        Password for data download. (default=None)
+        Path to directory to download data to.
+    user : str or NoneType
+        User name, raises an error if user not supplied. (default=None)
+    password : str or NoneType
+        User email, raises an error if not supplied. (default=None)
     file_type : str
         File format for Madrigal data. (default='hdf5')
     kindat : str
         Madrigal KINDAT code, specifies an experiment for the specified
         instrument.  May be a single value, blank (all), or a comma-delimited
         list. (defaut='')
 
-    Notes
-    -----
+    Raises
+    ------
+    ValueError
+        If user or password are not supplied
+
+    Note
+    ----
```

Comment:
Checking in since I believe that user names previously needed the '+' in between.

Reply:
That was an old convention, which isn't necessary.